cmd/cmd_synctarget.go

```diff
@@ -0,0 +1,156 @@
+					Schema:              c.String("schema"),
+					Table:               c.String("table"),
+				}
+				out, err := client.SyncTargetCreate(ctx, ac.Auth, input)
+				if err != nil {
+					return err
+				}
+				printlnif(c, out.Message(), false)
+				return nil
+			}),
+		},
+		{
+			Name:  "delete",
+			Usage: "Delete a sync target.",
+			Flags: []cli.Flag{
+				orgFlag(),
+				syncTargetFlag(),
+			},
+			Action: cliAction(func(c *cli.Context, ac appcontext.AppContext, ctx context.Context) error {
+				input := client.SyncTargetDeleteInput{
+					OpaqueId:      getSyncTargetFlagOrArg(c),
+					OrgIdentifier: getOrgFlag(c, ac.Prefs),
+				}
+				out, err := client.SyncTargetDelete(ctx, ac.Auth, input)
+				if err != nil {
+					return err
+				}
+				printlnif(c, out.Message, false)
+				return nil
+			}),
+		},
+		{
+			Name:  "list",
+			Usage: "List all sync targets for the given organization.",
+			Flags: []cli.Flag{
+				orgFlag(),
+				formatFlag(),
+			},
+			Action: cliAction(func(c *cli.Context, ac appcontext.AppContext, ctx context.Context) error {
+				input := client.SyncTargetListInput{
+					OrgIdentifier: getOrgFlag(c, ac.Prefs),
+				}
+				out, err := client.SyncTargetList(ctx, ac.Auth, input)
+				if err != nil {
+					return err
+				}
+				printlnif(c, out.Message(), true)
+				return getFormatFlag(c).WriteCollection(c.App.Writer, out)
+			}),
+		},
+		{
+			Name:  "update",
+			Usage: "Update a sync target",
+			Flags: []cli.Flag{
+				orgFlag(),
+				syncTargetFlag(),
+				periodFlag(),
+				schemaFlag(),
+				tableFlag(),
+			},
+			Action: cliAction(func(c *cli.Context, ac appcontext.AppContext, ctx context.Context) error {
+				input := client.SyncTargetUpdateInput{
+					OpaqueId:      getSyncTargetFlagOrArg(c),
+					OrgIdentifier: getOrgFlag(c, ac.Prefs),
+					Period:        c.Int("period"),
+					Schema:        c.String("schema"),
+					Table:         c.String("table"),
+				}
+				out, err := client.SyncTargetUpdate(ctx, ac.Auth, input)
+				if err != nil {
+					return err
+				}
+				printlnif(c, out.Message(), true)
+				return nil
+			}),
+		},
+		{
+			Name:  "update-creds",
+			Usage: "Update credentials for a sync target",
+			Flags: []cli.Flag{
+				orgFlag(),
+				syncTargetFlag(),
+				&cli.StringFlag{
+					Name:    "user",
+					Aliases: s1("u"),
+					Usage:   "Database username",
+				},
+				passwordFlag(),
+			},
+			Action: cliAction(func(c *cli.Context, ac appcontext.AppContext, ctx context.Context) error {
+				input := client.SyncTargetUpdateCredsInput{
+					OpaqueId:      getSyncTargetFlagOrArg(c),
+					OrgIdentifier: getOrgFlag(c, ac.Prefs),
+					Username:      c.String("user"),
+					Password:      c.String("password"),
+				}
+				out, err := client.SyncTargetUpdateCreds(ctx, ac.Auth, input)
+				if err != nil {
+					return err
+				}
+				printlnif(c, out.Message(), true)
+				return nil
+			}),
+		},
+		{
+			Name:  "sync",
+			Usage: "Sync a sync target",
```

Comment:
Trigger a sync to the sync target. The sync will happen at the earliest possible moment since the last sync, no matter how long the configured period is on the sync target.